src/main/java/ca/serum390/godzilla/domain/billOfMaterial/BomEntry.java

```diff
@@ -0,0 +1,27 @@
+package ca.serum390.godzilla.domain.billOfMaterial;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@Builder
+@ToString
+@Table("bill_of_material")
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class BomEntry {
+    
+    @Id
+    private Integer item_name;
```

Comment:
Why you are using Integer in some places and int is some other places?

Reply:
changed all to integer. the reason is to use the word integer. for repository.